libs/model/src/aggregates/user/Xp.projection.ts

```diff
@@ -594,6 +596,27 @@ export function Xp(): Projection<typeof schemas.QuestEvents> {
         if (!action_meta) return;
         await recordXpsForQuest(user_id, payload.created_at, [action_meta]);
       },
+      MembershipsRefreshed: async ({ payload }) => {
+        const action_metas = await getQuestActionMetas(
+          payload,
+          'MembershipsRefreshed',
+        );
+        await Promise.all(
```

Comment:
What's the approximate upper limit on the number of memberships included here?

Reply:
I'm going to simplify the event schema to just include ids and a flag, but in theory we should get a few "creates" on each refresh (from output middleware), and just the "updates" representing changes in balances that would change the existing membership => will have to review this case to just emit the deltas in memberships